CMP1903_A1_2324/Die.cs

```diff
@@ -20,7 +20,8 @@ internal class Die
         //Property
         //rolled value property to store what the die rolled
         public int rolledvalue = 0;
```

Comment:
Better to use encapsulation for more marks, so u would do the whole get set thing instead of just having a public variable

Reply:
encapsulation utilised